string robot_name
int32 planner_id
Waypoint[] path
Waypoint[] occupied_positions
Waypoint start_waypoint
Waypoint goal_waypoint